netbox/extras/scripts.py

```diff
@@ -500,12 +500,16 @@ def _log(self, message, obj=None, level=LogLevelChoices.LOG_DEFAULT):
                 'time': timezone.now().isoformat(),
                 'status': level,
                 'message': str(message),
+                'obj': str(obj) if obj else None,
+                'url': obj.get_absolute_url() if hasattr(obj, 'get_absolute_url') else None,
             })
 
             # Record to the system log
             if obj:
                 message = f"{obj}: {message}"
-            self.logger.log(LogLevelChoices.SYSTEM_LEVELS[level], message)
+                self.logger.log(LogLevelChoices.SYSTEM_LEVELS[level], f"{obj}: {message}")
+            else:
+                self.logger.log(LogLevelChoices.SYSTEM_LEVELS[level], message)
```

Comment:
This appears to be redundant to the line above (which is already prefixing the message with `{obj}: `).